static/js/tools/scatter2/plot_options.tsx

```diff
@@ -30,119 +30,106 @@ import { Container, Row, Col } from "reactstrap";
-                          ? 1e10
-                          : place.value.upperBound
-                      }
-                    />
-                  </FormGroup>
-                </Col>
-              </Row>
-              <Row>
-                <Col xs="3">Plot Options</Col>
-                <Col>
-                  <Button
-                    id="swap-axes"
-                    size="sm"
-                    color="light"
-                    onClick={() => swapAxes(x, y)}
-                  >
-                    Swap X and Y axes
-                  </Button>
-                </Col>
-                <Col>
-                  <FormGroup check>
-                    <Label check>
-                      <Input
-                        id="per-capita-x"
-                        type="checkbox"
-                        checked={x.value.perCapita}
-                        onChange={(e) => checkPerCapita(x, e)}
-                      />
-                      Plot X-axis per capita
-                    </Label>
-                  </FormGroup>
-                  <FormGroup check>
-                    <Label check>
-                      <Input
-                        id="per-capita-y"
-                        type="checkbox"
-                        checked={y.value.perCapita}
-                        onChange={(e) => checkPerCapita(y, e)}
-                      />
-                      Plot Y-axis per capita
-                    </Label>
-                  </FormGroup>
-                </Col>
-                <Col>
-                  <FormGroup check>
-                    <Label check>
-                      <Input
-                        id="log-x"
-                        type="checkbox"
-                        checked={x.value.log}
-                        onChange={(e) => checkLog(x, e)}
-                      />
-                      Plot X-axis on log scale
-                    </Label>
-                  </FormGroup>
-                  <FormGroup check>
-                    <Label check>
-                      <Input
-                        id="log-y"
-                        type="checkbox"
-                        checked={y.value.log}
-                        onChange={(e) => checkLog(y, e)}
-                      />
-                      Plot Y-axis on log scale
-                    </Label>
-                  </FormGroup>
-                </Col>
-              </Row>
-            </Container>
-          </Collapse>
+        <Row className="plot-options-row">
+          <Col sm={2}>Filter by population:</Col>
+          <Col>
+            <FormGroup check>
+              <Input
+                type="number"
+                onChange={(e) => selectLowerBound(place, e)}
+                value={place.value.lowerBound}
+              />
+            </FormGroup>
+          </Col>
+          <Col sm="auto">to</Col>
+          <Col>
+            <FormGroup check>
+              <Input
```

Comment:
would be nice to limit the width of these two inputs to a fixed / max width, instead of taking up the whole row on a very big monitor

Reply:
done